terraform/modules/container-definition/main.tf

```diff
@@ -17,8 +17,8 @@ output "json_format" {
     logConfiguration = {
       logDriver = "splunk",
       options = {
-        env               = "GOVUK_APP_NAME",
-        tag               = "image_name={{.ImageName}} container_name={{.Name}} container_id={{.FullID}}",
+        env               = "GOVUK_APP_NAME,GOVUK_ENVIRONMENT,GOVUK_WORKSPACE"
+        tag               = "image_name={{.ImageName}} container_name={{.Name}} container_id={{.FullID}} container_image_id={{.ImageFullID}}",
```

Comment:
I'm not sure we need `container_image_id` since `container_id` is the same value truncated to 12 chars ([doc](https://docs.docker.com/config/containers/logging/log_tags/)).

Reply:
good call missed this - was using it to test